exercises/strings/capitalize/capitalize.js

```diff
@@ -12,10 +12,15 @@
 
 function capitalize(num) {
   // This is your job. :)
+  let firstcharactercap= num.slice(0,1).toUpperCase();
+  let restofnum= num.slice(1);
+  return firstcharactercap + restofnum
 }
 
 if (require.main === module) {
   console.log('Running sanity checks for capitalize:');
+  console.log(capitalize('genesis'))
+  console.log(capitalize('mateo'))
```

Comment:
Consider adding negative tests, too, like `capitalize('Hello')`. By "negative" test I mean testing inputs that you _don't_ except to have a certain behavior.